pkg/apis/events/v1alpha1/scheduler_types.go

```diff
@@ -21,6 +21,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
+	"fmt"
```

Comment:
import formatter

Reply:
Is there a package called "formatter"? I can't find it. And I also checked the code in eventing and knative-gcp, we usually use "fmt"

Reply:
ohh no sorry. Just that you format the imports... the single ones, like "fmt" go in a group at the beginning.... For example: https://github.com/google/knative-gcp/blob/master/cmd/pubsub/publisher/main.go#L20